components/content/Command/AbstractCommand.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Joomla\Component\Content\Command;
+
+use Joomla\Command\CommandInterface;
+use Joomla\ORM\Definition\Locator\Locator;
+use Joomla\ORM\Definition\Locator\Strategy\RecursiveDirectoryStrategy;
+use Joomla\ORM\Repository\Repository;
+use Joomla\Renderer\Renderer;
+
+abstract class AbstractCommand implements CommandInterface
+{
+	/** @var Repository  */
+	protected $repository;
+
+	public function __construct()
+	{
+		$locator = new Locator([
+			new RecursiveDirectoryStrategy(dirname(__DIR__) . '/Entity'),
+		]);
+
+		$this->repository = new Repository('Article', $locator);
+	}
+}
```

Comment:
The construction of the Repository has to be changed. Most likely should go into the DIC.